Cleanup of factory reference manual section (#628)
foundriesio/docs: source/reference-manual/factory/factory-definition.rst

notify:
 email:
  users: ``<email_1>,<email_2>,<...>``

>       **Required:** A Comma separated list of email addresses.

Reviewer: I still think the old content is relevant. This lists the emails to notify a build has passed/fail.

>       **Required:** A Comma separated list of email addresses.

Author: ```suggestion
      **Required:** A Comma separated list of email addresses to email after each CI build.
```

           params:
             EXAMPLE_VARIABLE_1: hello_world
           machines:
             - imx8mm-lpddr4-evk
           image_type: lmp-factory-image
           ref_options:
            refs/heads/main:
              machines:
                - raspberrypi4-64
              params:
                IMAGE: lmp-factory-image
                EXAMPLE_VARIABLE_1: foo
           tagging:
             refs/heads/main:
               - tag: postmerge
             refs/heads/feature1:
               - tag: feature1
           mfg_tools:

>              - machine: imx8mmevk

Reviewer: ```suggestion
             - machine: imx8mm-lpddr4-evk
```

             EXAMPLE_VARIABLE_1: hello_world
           machines:

>              - imx8mmevk

Reviewer: Current machine name:
```suggestion
             - imx8mm-lpddr4-evk
```

>            image_type: lmp-base-console-image

Reviewer: Hmm not really. `lmp-factory-image` is still the image to go for factories. `lmp-base-console-image` is the one from LmP.

>            image_type: lmp-base-console-image

Author: ```suggestion
           image_type: lmp-factory-image
```

           ref_options:
            refs/heads/main:
              machines:
                - raspberrypi4-64
              params:

>                 IMAGE: lmp-mini-image

Reviewer: ```suggestion
                IMAGE: lmp-factory-image
```

lmp:
 preloaded_images:
  enabled: ``<true|false>``
      **Optional:** Whether to preload Docker images into the system-image as part of a platform build.

      **Default:** ``false``

      **Inherits:** ``containers``

  shortlist: ``<app1>,<app2>,<...>``
      **Optional:** Comma separated list of apps to preload.
      If not specified, or its value is empty, then all of a Target's apps are preloaded.

      **Default:**  None

 params:
  EXAMPLE_VARIABLE_1: ``<value>``
      **Optional:** Define an arbitrary environment variable to be passed into the LmP build.
      You can define as many as you want.

      **Default:** This variable is user defined and does not exist unless instantiated.

 machines:|br| ``- <machine_1>`` |br| ``- <machine_2>``
      **Required**: Specify the list of :ref:`Supported LmP Machines <ref-linux-supported>` to build for, using the ``MACHINE`` name.
      A Factory's subscription is generally only for a single machine.

      **Default**: Set during :ref:`gs-signup`.

     .. note::
        
        The CI is configured to decline changes to the ``machines:`` parameter.

>         If needed, ask a support engineer to update the machine definition for your Factory.

Reviewer: ```suggestion
        If needed, ask a support engineer to update the machine definition for your Factory.
```
We need a link to support.foundries.io here

Author: ```suggestion
        If needed, `ask a support engineer <support.foundries.io>`_ to update the machine definition for your Factory.
```

 image_type:``<lmp_image_type>``
      **Optional:** Set the LmP image type by recipe name.
      For example, ``lmp-mini-image``, ``lmp-base-console-image`` from meta-lmp_.

>       **Default:** ``lmp-base-console-image`` |br| 

Author: ```suggestion
      **Default:** ``lmp-factory-image`` |br| 
```

>       **Default:** ``lmp-base-console-image`` |br| 
>       from ``recipes-samples/images/lmp-base-console-image.bb`` in your ``meta-subscriber-overrides.git`` repo.

Reviewer: ```suggestion
      **Default:** ``lmp-base-console-image`` |br| 
      from ``recipes-samples/images/lmp-base-console-image.bb`` in your ``meta-subscriber-overrides.git`` repo.
```
This is wrong, please revert to the original.
base console comes from LmP - fast first target, however factory-config should point to the factory image which is included in meta-subscriber-overrides (there's no base console image there).
If it's not clear why, please let me know in Slack.

>       from ``recipes-samples/images/lmp-base-console-image.bb`` in your ``meta-subscriber-overrides.git`` repo.

Author: ```suggestion
      from ``recipes-samples/images/lmp-factory-image.bb`` in your ``meta-subscriber-overrides.git`` repo.
```

>       from ``recipes-samples/images/lmp-base-console-image.bb`` in your ``meta-subscriber-overrides.git`` repo.

Reviewer: ```suggestion
      from ``recipes-samples/images/lmp-factory-image.bb`` in your ``meta-subscriber-overrides.git`` repo.
```

ref_options:
  refs/heads/``<branch>``:
      **Optional:** Override options when specific git references are updated

      **Example:**

      .. code-block:: yaml

	      # In the below example, when the branch named "feature1" is built by our
	      # CI system, it will have its option values for "machine" and
	      # "params" overriden by what is specified after "refs/heads/feature1:".
	      # In the "feature1" build, IMAGE will now equal "lmp-mini-image" rather than

> 	      # "lmp-base-console-image" as initially defined.

Reviewer: ```suggestion
	      # "lmp-factory-image" as initially defined.
```

               lmp:
                 params:

>                   IMAGE: lmp-base-console-image

Reviewer: ```suggestion
                  IMAGE: lmp-base-console-image
```
Revert

>                   IMAGE: lmp-base-console-image

Author: ```suggestion
                  IMAGE: lmp-factory-image
```

>                   IMAGE: lmp-base-console-image

Author: ```suggestion
                  IMAGE: lmp-factory-image
```

                machines:

>                   - imx8mmevk

Author: ```suggestion
                  - imx8mn-ddr4-evk
```

                ref_options:
                  refs/heads/devel:
                    machines:

>                       - raspberrypi4-64

Author: ```suggestion
                      - imx8mn-ddr4-evk
```

                    params:
                      IMAGE: lmp-mini-image

 tagging:
  refs/heads/``<branch>``:|br| ``-tag: <tag>``
      **Optional:** Control how OTA_LITE tags are handled. See
      :ref:`ref-advanced-tagging` for more details.

 mfg_tools:|br| ``- machine: <machine>``
      **Optional:** Do an OE build to produce manufacturing tooling for a given ``MACHINE``.
      This is used to facilitate the manufacturing process, and to ensure secure boot on devices.

>       Currently, only NXP tools are supported.

Author: ```suggestion
      Currently, only NXP® tools are supported.
```

containers:
 preloaded_images:
  enabled: ``<true|false>``
      **Optional:** Whether to preload Docker images into the system-image as part of a containers build.

      **Default:** ``false``

  shortlist: ``<app1>,<app2>,<...>``
      **Optional:** Comma separated list of apps to preload.
      If it is not specified or its value is empty, then all Target's apps are preloaded.

      **Default:**  None

 platforms:|br| ``- arm`` |br| ``- arm64`` |br| ``- amd64``
      **Optional:** Specify a list of architectures to build containers for.
      Containers are only built for the specified list.

>       **Default:** Variable, set during Signup. 

Reviewer: ```suggestion
      **Default:** Variable, set during Signup. 
```
I didn't know about this one, for me it still defaults to the 3 archs.

>       **Default:** Variable, set during Signup. 

Author: ```suggestion
      **Default:** ``arm,arm64,amd64``. 
```
